src/audio/host.c

```diff
@@ -447,6 +448,21 @@ static void host_buffer_cb(void *data, uint32_t bytes)
 		MIN(avail_bytes, hd->dma_buffer->free) :
 		MIN(hd->dma_buffer->avail, free_bytes);
 
+	ret = dma_get_attribute(hd->dma, DMA_ATTR_COPY_ALIGNMENT,
```

Comment:
It would be better to get it once e.g. in host_new() and save it in host component private data.

Reply:
I've made it in host_params().